src/Dfe.PlanTech.AzureFunctions/Mappings/RecommendationChunkUpdater.cs

```diff
@@ -29,17 +29,16 @@ public override MappedEntity UpdateEntityConcrete(MappedEntity entity)
     private void RemoveOldAssociatedChunkContents(RecommendationChunkDbEntity existingRecommendationChunk)
     {
         var contentsToRemove = Db.RecommendationChunkContents
-            .Where(content => content.RecommendationChunkId == existingRecommendationChunk.Id)
-            .ToList();
+            .Where(content => content.RecommendationChunkId == existingRecommendationChunk.Id);
 
-        Db.RecommendationChunkContents.RemoveRange(contentsToRemove);
+        Db.RecommendationChunkContents.RemoveRange(Db.RecommendationChunkContents
+            .Where(content => content.RecommendationChunkId == existingRecommendationChunk.Id));
     }
 
     private void RemoveOldAssociatedChunkAnswers(RecommendationChunkDbEntity existingRecommendationChunk)
     {
         var answersToRemove = Db.RecommendationChunkAnswers
-            .Where(content => content.RecommendationChunkId == existingRecommendationChunk.Id)
-            .ToList();
+            .Where(content => content.RecommendationChunkId == existingRecommendationChunk.Id);
 
         Db.RecommendationChunkAnswers.RemoveRange(answersToRemove);
```

Comment:
maybe stay consistent within the class and don't assign the var, but just remove instead

Reply:
I like keeping it separate for readability, especially when there's a longer query.